txn_writer.go

```diff
@@ -152,7 +152,7 @@ func (sw safeWriter) writeAllSafe(forceVendor bool) error {
 
 	if writeM {
 		if _, err := os.Stat(mpath); err == nil {
-			// move out the old one
```

Comment:
yeah, i did see those, but they weren't what i was referring to - [some](https://golang.org/src/go/build/build.go?#L1387) [examples](https://golang.org/src/go/build/build.go?#L1034). (cruft happens, of course).

that's fine, though, i can modify my habits accordingly.